crates/tests/nightly_component_client/build.rs

```diff
@@ -0,0 +1,23 @@
+use std::fs::*;
+use std::io::prelude::*;
+use std::process::*;
+use windows_bindgen::*;
+
+fn main() {
+    std::fs::create_dir_all(".windows/winmd").unwrap();
+
+    // I would have expected Cargo to wait until the component crate (which is a dependency) is built
```

Comment:
I believe if you mark the component as a build dependency (and not just a regular dependency) this won't happen.

Reply:
Ah - that's a relief! 🤣